contracts/contracts/ERC677/link_token.cairo

```diff
@@ -0,0 +1,38 @@
+%lang starknet
```

Comment:
While this contract is ERC677, or to be exact it should be an ERC677Mintable token, it is not part of the ERC677 protocol so please remove it from the ERC677 folder.

Move it to the parent folder.